lib/declarativedeclarativecontext.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include "declarativedeclarativecontext_p.h"
 
-#include <QDeclarativeView>
+#include <QQuickWidget>
 
 DeclarativeDeclarativeContext::DeclarativeDeclarativeContext(QObject *parent)
```

Comment:
At some point we should rename the classes that wrap QQml... classes now.
I.e. this one is no longer wrapping QDeclarativeContext but QQmlContext, so a more appropriate name would be DeclarativeQmlContext